doc/ref_arch/openstack/chapters/chapter08.rst

```diff
@@ -1,35 +1,1698 @@
+
+   * - Scenarios
+     - Iterations
+   * - Authenticate.validate_nova
+     - 10
+   * - NovaKeypair.create_and_delete_keypair
+     - 10
+   * - NovaKeypair.create_and_list_keypairs
+     - 10
+   * - NovaServers.boot_and_bounce_server
+     - 10
+   * - NovaServers.boot_and_delete_server
+     - 10
+   * - NovaServers.boot_and_list_server
+     - 10
+   * - NovaServers.boot_and_rebuild_server
+     - 10
+   * - NovaServers.snapshot_server
+     - 10
+   * - NovaServers.boot_server_from_volume
+     - 10
+   * - NovaServers.boot_server
+     - 10
+   * - NovaServers.list_servers
+     - 10
+   * - NovaServers.resize_server
+     - 10
+   * - NovaServers.boot_and_live_migrate_server
+     - 10
+   * - NovaServers.boot_server_attach_created_volume_and_live_migrate
+     - 10
+   * - NovaServers.boot_server_from_volume_and_live_migrate
+     - 10
+   * - NovaKeypair.boot_and_delete_server_with_keypair
+     - 10
+   * - NovaServers.boot_server_from_volume_and_delete
+     - 10
+   * - NovaServers.pause_and_unpause_server
+     - 10
+   * - NovaServers.boot_and_migrate_server
+     - 10
+   * - NovaServers.boot_server_and_list_interfaces
+     - 10
+   * - NovaServers.boot_and_get_console_url
+     - 10
+   * - NovaServers.boot_server_and_attach_interface
+     - 10
+   * - NovaServers.boot_server_attach_volume_and_list_attachments
+     - 10
+   * - NovaServers.boot_server_associate_and_dissociate_floating_ip
+     - 10
+   * - NovaServers.boot_and_associate_floating_ip
+     - 10
+   * - NovaServerGroups.create_and_delete_server_group
+     - 10
+   * - NovaServerGroups.create_and_get_server_group
+     - 10
+   * - NovaServerGroups.create_and_list_server_groups
+     - 10
+   * - Quotas.nova_update
+     - 10
+
+Orchestration - Heat API benchmarking
+'''''''''''''''''''''''''''''''''''''
+
+`Functest
+rally_full_cntt <http://artifacts.opnfv.org/functest/KDBNITEN317M/functest-opnfv-functest-benchmarking-cntt-wallaby-rally_full_cntt-run-5/rally_full_cntt/rally_full_cntt.html>`__:
+
+.. table:: Heat API benchmarking
+   :widths: auto
+
+   ============================================= ==========
+   Scenarios                                     Iterations
+   ============================================= ==========
+   Authenticate.validate_heat                    10
+   HeatStacks.create_and_delete_stack            10
+   HeatStacks.create_and_list_stack              10
+   HeatStacks.create_update_delete_stack         10
+   HeatStacks.create_check_delete_stack          10
+   HeatStacks.create_suspend_resume_delete_stack 10
+   HeatStacks.list_stacks_and_resources          10
+   ============================================= ==========
+
+Dataplane benchmarking
+^^^^^^^^^^^^^^^^^^^^^^
+
+`Functest Benchmarking
+CNTT <https://git.opnfv.org/functest/tree/docker/benchmarking-cntt/testcases.yaml?h=stable%2Fwallaby>`__
+offers two benchmarking dataplane test cases leveraging on:
+
+-  `VMTP <http://vmtp.readthedocs.io/en/latest>`__
```

Comment:
Should we add VMTP to Terminology? also I think I missed DHCP? If using CNTT then it should also be added to Terminology